public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/appInsights.json

```diff
@@ -0,0 +1,2645 @@
+            "resultFormat": "time_series"
+          },
+          "azureMonitor": {
+            "aggOptions": [],
+            "aggregation": "Count",
+            "allowedTimeGrainsMs": [60000, 300000, 900000, 1800000, 3600000, 21600000, 43200000, 86400000],
+            "dimensionFilter": "*",
+            "dimensionFilters": [],
+            "metricDefinition": "Microsoft.Insights/components",
+            "metricName": "exceptions/browser",
+            "metricNamespace": "microsoft.insights/components",
+            "resourceGroup": "$rg",
+            "resourceName": "$res",
+            "timeGrain": "auto",
+            "timeGrains": [],
+            "top": "50"
+          },
+          "azureResourceGraph": {
+            "resultFormat": "table"
+          },
+          "insightsAnalytics": {
+            "query": "",
+            "resultFormat": "time_series"
+          },
+          "queryType": "Azure Monitor",
+          "refId": "A",
+          "subscription": "$sub",
+          "subscriptions": [
+            "4dc2cd39-7a89-43d8-bebe-8bb501359891",
+            "39a6a513-91ca-4958-aa57-8374048b16ab",
+            "f7152080-b4e8-47ee-9c85-7f1d0e6b72dc"
+          ]
+        }
+      ],
+      "title": "Browser exceptions",
+      "type": "timeseries"
+    }
+  ],
+  "refresh": false,
+  "schemaVersion": 30,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "current": {
+          "selected": false,
+          "text": "Azure Monitor",
+          "value": "Azure Monitor"
+        },
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": "Datasource",
+        "multi": false,
+        "name": "ds",
+        "options": [],
+        "query": "grafana-azure-monitor-datasource",
+        "queryValue": "",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "type": "datasource"
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "Test/Demo - One Observability",
+          "value": "263177d8-32b8-465a-b939-be00c75e88c5"
+        },
+        "datasource": "${ds}",
+        "definition": "Subscriptions()",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": "Subscription",
+        "multi": false,
+        "name": "sub",
+        "options": [],
+        "query": "Subscriptions()",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "kaprince-grafana",
```

Comment:
I'm guessing this is is also your particular selection here, maybe we can make it current: {}